climada/hazard/trop_cyclone.py

```diff
@@ -270,13 +270,15 @@ def from_tracks(
 
         if ignore_distance_to_coast:
             # Select centroids with lat <= max_latitude
-            coastal_idx = (np.abs(centroids.lat) <= max_latitude).nonzero()[0]
+            [idx_centr_filter] = (np.abs(centroids.lat) <= max_latitude).nonzero()
```

Comment:
The variable name used to be misleading because it was describing a filtered list of centroids, and filtering was not only done based on the property of being a "coastal" centroid or not. The "coastal" criterion was only one of several (optional) criteria.